README.rst

```diff
@@ -6,59 +6,43 @@ Taisei
 Introduction
 ------------
 
-Taisei is an open clone of the Tōhō Project series. Tōhō is a one-man project of
-shoot-em-up games set in an isolated world full of Japanese folklore.
+Taisei Project is an open clone of the Tōhō Project series. Tōhō is a one-man
+project of shoot-em-up games set in an isolated world full of Japanese folklore.
 
-Installation
-------------
-
-Dependencies
-^^^^^^^^^^^^
-
--  OpenGL >= 3.3 or OpenGL ES >= 3.0 or OpenGL ES >= 2.0 (with some extensions)
--  SDL2 >= 2.0.6
--  SDL2_mixer >= 2.0.4
--  freetype2
--  libpng >= 1.5.0
--  libwebpdecoder >= 0.5 or libwebp >= 0.5
--  libzip >= 1.2
--  opusfile
--  zlib
+For gameplay instructions, read `this <doc/GAME.rst>`__.
 
-Optional:
+For the story, read `this <doc/STORY.txt>`__. (Warning: spoilers!)
 
--  OpenSSL (for a better SHA-256 implementation; used in shader cache)
--  SPIRV-Cross >= 2019-03-22 (for OpenGL ES backends)
--  libshaderc (for OpenGL ES backends)
-
-Build-only dependencies
-^^^^^^^^^^^^^^^^^^^^^^^
+Installation
+------------
 
--  Python >= 3.5
--  meson >= 0.48.0 (build system; >=0.49.0 recommended)
+You can find precompiled binaries of the complete game on the
+`Releases <https://github.com/taisei-project/taisei/releases>`__ page on
+GitHub, available for Windows (x86/x64), Linux, and macOS. An experimental
+build for Nintendo Switch (homebrew) also exists (use at your own risk).
 
-Optional:
+Source Code & Development
+-------------------------
 
--  docutils (for documentation)
+You can obtain the source code of the stable releases from the
+`Releases <https://github.com/taisei-project/taisei/releases>`__ page on
+GitHub, alongside the binaries.
 
-Obtaining the source code
-^^^^^^^^^^^^^^^^^^^^^^^^^
+**Important:** *do not* grab GitHub's auto-generated source archives, as those
+do not contain the required submodules for compiling the project. This only
+applies to versions v1.3 and above.
 
-Stable releases
-"""""""""""""""
+You can also grab the latest code from git using the following commands:
 
-You can find the source tarballs at the
-`Releases <https://github.com/taisei-project/taisei/releases>`__ section on
-Github. **Do not** grab Github's auto-generated source archives, those do not
-contain the required submodules. This only applies for versions v1.3 and above.
+::
 
-Latest code from git
-""""""""""""""""""""
+    git clone https://github.com/taisei-project/taisei
```

Comment:
It's probably better to include the `--recurse-submodules` option in this command line, and explain its importance below (as well as how to update the submodules).